ui/src/logs/actions/index.ts

```diff
@@ -1,13 +1,22 @@
-import {Source, Namespace, TimeRange} from 'src/types'
+import {Source, Namespace, TimeRange, QueryConfig} from 'src/types'
 import {getSource} from 'src/shared/apis'
 import {getDatabasesWithRetentionPolicies} from 'src/shared/apis/databases'
+import {buildHistogramQueryConfig} from 'src/logs/utils'
 import {getDeep} from 'src/utils/wrappers'
+import buildQuery from 'src/utils/influxql'
+import {executeQueryAsync} from 'src/logs/api'
+import {LogsState} from 'src/types/localStorage'
```

Comment:
would you consider grouping all lines where types are imported, and separating them by an empty line, at the end of this list of imports? personally, i find it easier to read imports when they're grouped